### DISCUSSION TITLE
开发者例会议题收集（2023-01-10）

### DISCUSSION THREAD

**Discussion by chuntaojun** [polarismesh/polaris#890] [Meeting]
# 开发者例会议题收集（2023-01-10）

## 会议信息

andrewshan(单家骏) 邀请您参加腾讯会议
会议主题：PolarisMesh社区双周例会
会议时间：2022/12/12 20:00-20:30 (GMT+08:00) 中国标准时间 - 北京
重复周期：2022/12/12-2023/03/21 20:00-20:30, 每两周 (周二)

点击链接入会，或添加至会议列表：
https://meeting.tencent.com/dm/bi26r1VIAx2y

#腾讯会议：825-9154-7185

## 会议录屏

主题: PolarisMesh社区双周例会
日期: 2023-01-10 19:50:03
录制文件：https://meeting.tencent.com/v2/cloud-record/share?id=d393ec83-62b4-41be-b011-4d54afab3d6a&from=3

**Comment by shenqidebaozi** [comment]
议题名称: kratos 和 Polaris 集成
主讲人: 包子

**Comment by andrewshan** [comment]
北极星的新版本熔断功能介绍
